RoboJackets/robocup-software
# Test play improvements

## Description
See issue below for description.

## Associated Issue
Resolves #1896 

## Steps to test
### Test Case 1
~~1. note the required format of the plays (see the plays set close to the top of gameplay node in its constructor).~~
2. go to testing tab in ui, ~~add play in that format.~~
3. click the desired play in the top table, and hit run.
4. all of them will work (except lineup because it's broken rn)

I did not implement the next button. It would be easy to set up later though. 
I also did not use the python local param server (I commented it out, as I think it's overengineering for now, but it could be important later)
Later we can easily just at launch populate all valid plays into the selected plays list.

Also this pr deletes some unused ui stuff.
But at this point because it is functional, this pr is closed to new features.

Expected result: can change test play at runtime

Someone should double check trying to run invalid formats, non-existent plays, etc.. The code should handle all those cases.

Also you can use `ros2 param set /gameplay_node test_play formatted_name`
